recipes/cpputest/all/conanfile.py

```diff
@@ -0,0 +1,100 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class CppUTestConan(ConanFile):
+    name = "cpputest"
+    description = \
+"CppUTest is a C /C++ based unit xUnit test framework for unit testing and for test-driving your code." \
+"It is written in C++ but is used in C and C++ projects and frequently" \
+"used in embedded systems but it works for any C/C++ project."
+    license = "BSD-3-Clause License"
+    topics = ("conan", "testing", "unit-testing")
+    homepage = "http://cpputest.github.io"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "verbose": [True, False],
+        "use_std_c_lib": ["ON", "OFF"],
+        "use_std_cpp_lib": ["ON", "OFF"],
+        "use_cpp11": ["ON", "OFF"],
+        "detect_mem_leaks": ["ON", "OFF"],
+        "extensions": ["ON", "OFF"],
+        "longlong": ["ON", "OFF"],
+        "coverage": ["ON", "OFF"],
+        "tests": ["ON", "OFF"],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": False,
+        "verbose": False,
+        "use_std_c_lib": "ON",
+        "use_std_cpp_lib": "ON",
+        "use_cpp11": "ON",
+        "detect_mem_leaks": "ON",
+        "extensions": "ON",
+        "longlong": "ON",
+        "coverage": "OFF",
+        "tests": "OFF",
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
```

Comment:
Shoud it not be conditional on user_cpp option?

Reply:
Please could you explain what I need to do?

Reply:
@prince-chrismc Hi Chris, do I just need to delete these two lines:

```python
        del self.settings.compiler.libcxx
        del self.settings.compiler.cppstd
```

If not, how could I find out what I need to do instead, please?

Reply:
WIP.

```suggestion
        if not self.option.
        del self.settings.compiler.libcxx
        del self.settings.compiler.cppstd
```

Reply:
Ah right - Thanks - trying it out now.